docs/source/api/cacheassignmentgroup.rst

```diff
@@ -0,0 +1,239 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-cacheassignmentgroups:
+
+***************
+``cacheassignmentgroups``
+***************
```

Comment:
@ocket8888 do you have a tool you like for linting these files?

Reply:
I just build the docs - it would warn you about this. The only thing it wouldn't warn about is spaces vs tabs.

The relevant sublime text build I use to highlight errors/warnings in my editor:
```jsonc
{
	"name": "Build Documentation",
	"working_dir": "$folder/docs", // Needs to be in a project file at the repository root, otherwise edit this to be the path to the repo's "docs" directory
	"cmd": ["make", "SPHINXOPTS='-j 4'"],
	"file_regex": "(/(?:.+)\\.rst):(\\d+): (?:WARNING|ERROR): (.*)",
	"selector": "text.restructuredtext"
},
```